install/hst-install-debian.sh

```diff
@@ -1977,6 +1977,8 @@ command="sudo $HESTIA/bin/v-update-user-stats"
 $HESTIA/bin/v-add-cron-job 'admin' '20' '00' '*' '*' '*' "$command"
 command="sudo $HESTIA/bin/v-update-sys-rrd"
 $HESTIA/bin/v-add-cron-job 'admin' '*/5' '*' '*' '*' '*' "$command"
+command="sudo $HESTIA/bin/v-update-letsencrypt-ssl"
+$HESTIA/bin/v-add-cron-job 'admin' '30' '07' '*' '*' '*' "$command"
```

Comment:
Random times should be used to limit DDOS effect to LE servers

Reply:
Function generate_password() defined at func/main.sh

So should I duplicate this func at `hst-install-*.sh` if we need it to be used?

Reply:
Modifiing should be fine